docs/gallery/interactive/pyfar_filtering.ipynb

```diff
@@ -0,0 +1,281 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Filtering pyfar Signals\n",
+    "\n",
+    "Filtering signals is a very common task in acoustics signal processing and pyfar provides different options for this task that are explored in the following examples."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pyfar as pf\n",
+    "import numpy as np\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Filtering with `pyfar.dsp.filter`\n",
+    "\n",
+    "The *filter* module contains a variety of common filter functions including filter banks, bell, and shelve filters, as well as classic filter types such as low-passes and band-passes. See the `filter type` examples for an overview. Using the filter module is the simplest way to filter a pyfar Signal object. As the following example of a low-pass shows, the function in the filter module takes a Signal object as input and directly returns the filtered Signal\n",
```

Comment:
maybe add a link here to to pyfar doc somehow. 
and what means "See the `filter type` examples for an overview" Is there a link missing?

Reply:
Added the link and more clearly marked the 'to-be-inserted' link tot he filter types notebook.